test/unit/YieldTest.sol

```diff
@@ -11,21 +11,21 @@ contract UnitYieldTest is BaseTest {
   uint256 public constant MIN_VOUCHES = 3;
   uint256 public constant PROJECT_COUNT = 3;
 
-  function test_RevertWhenCycleNotReady() public {
```

Comment:
can probably just remove `test_RevertWhenCycleNotReady` test instead of commenting out

Reply:
are they passing now @hudsonhrh ?